src/libs/elektra/keymeta.c

```diff
@@ -609,6 +618,11 @@ ssize_t keySetMeta (Key * key, const char * metaName, const char * newMetaString
  * @note You must not delete the returned KeySet.
  * @note Adding a key with metadata to the KeySet is an error.
  *
+ * @post for the returned KeySet ks: keyGetMeta(key, metaName) ==
+ * ksLookupByName(ks, metaName)
+ * @post if @p key did not contain metadata, a new KeySet for the metadata will
```

Comment:
Simply write that there will be a KeySet for the metadata, it does not matter if and when it gets created